Mathlib/Topology/Algebra/Module/TransferInstance.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2025 Michael Rothgang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Rothgang
+-/
+module
+
+public import Mathlib.Analysis.Normed.Module.Basic
+public import Mathlib.Algebra.Module.TransferInstance
+public import Mathlib.Topology.Algebra.Module.Equiv
+
+/-!
+# Transfer algebraic structures across `Equiv`s
+
+In this file, we transfer a topological space and continuous linear equivalence structure
+across an equivalence.
+This continues the pattern set in `Mathlib/Algebra/Module/TransferInstance.lean`.
+-/
+
+@[expose] public section
+
+variable {R α β : Type*}
+
+namespace Equiv
+
+variable (e : α ≃ β)
+
+/-- Transfer a `TopologicalSpace` across an `Equiv` -/
+protected abbrev topologicalSpace (e : α ≃ β) : ∀ [TopologicalSpace β], TopologicalSpace α :=
```

Comment:
If I move the `TopologicalSpace beta` instance to the left of the colon, I get an error on line 50.

Reply:
It's because the proof has to become different (you're not identifying any more two functions acting on topological spaces, but two topological spaces). This works for instance with
```
      simp only [Equiv.topologicalSpace, ← @coinduced_symm]
      exact continuous_coinduced_rng
```

Reply:
Right! Thank you, fixed.